api/v1/verticadb_types.go

```diff
@@ -888,6 +901,12 @@ type SandboxStatus struct {
 	// +optional
 	// State of the current running upgrade in the sandbox
 	UpgradeState SandboxUpgradeState `json:"upgradeState,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=status
+	// +optional
+	// State of the sandbox. true means the sandbox is currently down
+	// and must not be restarted.
+	Shutdown bool `json:"shutdown"`
```

Comment:
This should be removed since you already have one in SubclusterStatus. If a sandbox is stopped, we should update all subclusters' status inside it.

Reply:
This is still useful:
- From the realdb script you can just check this field, instead of all the sandbox subclusters status, to know if the sandbox  shutdown state is being enforced.
- Updating `shutdown` in podfacts is done earlier. As soon as the user sets `spec.sandboxes[].shutdown` to true all the nodes in that subcluster should inherit that state and must not be restarted by the operator. This makes the operator more reactive to this change as it does not have to wait for the subcluster(s) update.

Simply put it does not hurt to have this.